pilot/pkg/bootstrap/istio_ca.go

```diff
@@ -112,14 +131,19 @@ var (
 
 	caRSAKeySize = env.RegisterIntVar("CITADEL_SELF_SIGNED_CA_RSA_KEY_SIZE", 2048,
 		"Specify the RSA key size to use for self-signed Istio CA certificates.")
-)
 
-type CAOptions struct {
-	// domain to use in SPIFFE identity URLs
-	TrustDomain    string
-	Namespace      string
-	Authenticators []authenticate.Authenticator
-}
+	//TODO: Likely to be removed and added to mesh config
+	externalCA = env.RegisterStringVar("EXTERNAL_CA", "",
+		"External CA Type").Get()
+
+	//TODO: Likely to be removed and added to mesh config
+	k8sSigner = env.RegisterStringVar("K8S_SIGNER", "",
```

Comment:
ditto, confused for users

Reply:
Any recommendations on what to add here? If the customer is using this feature, it is expected that they know which K8s Signer they want. This is an arbitrary string value.

Reply:
understood